tests/models/test_experts_int8.py

```diff
@@ -0,0 +1,25 @@
+# flake8: noqa
```

Comment:
Maybe this should be moved to `tests/quantization/test_experts_int8.py`

Reply:
Moved it to quantization sub dir